common/vm_set_nested_virtual.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Enable or disable nested virtualization on VM
+# Parameters:
+#   vm_nested_virt: true or false.
+# if set to true, full CPU virtualization will be exposed to the guest operating system
```

Comment:
The first letter should be capitalized as it is a new sentence. And the indent looks not good, try to put 4 spaces before `If` in this case.

Reply:
Fixed. Thanks.